fix(README): add some tips (#433)
GreptimeTeam/dashboard: README.md

- Visit `http://localhost:4000/dashboard`

### With Docker
⭐We recommend using docker-compose to setup both greptimedb and dashboard.
1. Use the `curl` command to get the `docker-compose` configuration file
```
curl -O https://raw.githubusercontent.com/greptimeteam/dashboard/main/docker/docker-compose.yml

> ```
> > Tips: If you encounter network-related problems when executing the `curl` commands. Download the `docker-compose.yml` file manually or simply copy and paste the [contents](https://github.com/GreptimeTeam/dashboard/blob/main/docker/docker-compose.yml) into an empty `yml` file on your machine or server, based on what you need.

Reviewer: ```suggestion
```
> Tips: If you encounter network-related problems, you could download the `docker-compose.yml` file manually or simply copy and paste the [contents](https://github.com/GreptimeTeam/dashboard/blob/main/docker/docker-compose.yml) into an empty `yml` file on your machine or server, based on what you need.
```

> > Tips: If you encounter network-related problems when executing the `curl` commands. Download the `docker-compose.yml` file manually or simply copy and paste the [contents](https://github.com/GreptimeTeam/dashboard/blob/main/docker/docker-compose.yml) into an empty `yml` file on your machine or server, based on what you need.

Reviewer: ```suggestion
> Tips: If you encounter network-related problems, you can download the `docker-compose.yml` file manually or simply copy and paste the [contents](https://github.com/GreptimeTeam/dashboard/blob/main/docker/docker-compose.yml) into an empty `yml` file on your machine or server, based on what you need.
```

- Visit `http://localhost:4000/dashboard`

### With Docker
⭐We recommend using docker-compose to setup both greptimedb and dashboard.
1. Use the `curl` command to get the `docker-compose` configuration file

> > Tips: If you are located in China and do not have access to an external network, you are likely to encounter network-related problems when executing the following commands.

Reviewer: ```suggestion
```

```
curl -O https://raw.githubusercontent.com/greptimeteam/dashboard/main/docker/docker-compose.yml
```

> > Solution: Download the `docker-compose.yml` file locally and transfer it to the VM or server using `SFTP`. Or just copy the contents of [this file](https://github.com/GreptimeTeam/dashboard/blob/main/docker/docker-compose.yml) and create the file in the virtual machine or server.

Reviewer: ```suggestion
> Tips: If you encounter network-related problems when executing the `curl` commands. Download the `docker-compose.yml` file manually or simply copy and paste the [contents](https://github.com/GreptimeTeam/dashboard/blob/main/docker/docker-compose.yml) into an empty `yml` file on your machine or server, based on what you need.
```

Author: You're welcome, thanks for the reply! I think this is a great suggestion on your part, and much more clearly articulated than its predecessor!👍🤝
